composer.json

```diff
@@ -15,6 +15,7 @@
     "ext-PDO": "*",
     "ext-json": "*",
     "ext-sockets": "*",
+    "colinmollenhour/credis": "^1.8.2",
```

Comment:
@shiftedreality this is not correct. Magento 2.2.0 has `1.6` for example. The 2.1.x version you referenced to was updated after early 2.2.x versions. This needs to be reverted. Reference: https://github.com/magento/magento-cloud/blob/2.2.0/composer.lock#L129